### ISSUE TITLE
Add sendText and operator << to Player and Playerset

### ISSUE THREAD

**Issue by GoogleCodeExporter** [macressler/bwapi#488]
```
The addition of sendText as a member of Player will allow the client to easily 
send private messages to specific players in the game (a lacking feature).

Similarly, this addition would also be made to Playerset so that messages can 
be sent to player groups.

This will allow the use of stuff like: Broodwar->allies().sendText("Hey!"), 
instead of using the unclear sendTextEx. It will also deprecate sendTextEx.

Then, we can put Player in a wrapper for overloading operator << like in 
Broodwar. The same operator << can easily be put into Playerset.

Code then becomes:
Broodwar->allies() << "Hey!" << endl;
someUnit->getPlayer() << "Your " << someUnit->getType() << " is in my way." << 
endl;

----
tl;dr

- Add sendText member to Player and Playerset.
- Add operator << overload to Playerset.
- Wrap Player to overload operator <<.
- Deprecate/repurpose sendTextEx.

```

Original issue reported on code.google.com by `AHeinerm` on 20 May 2013 at 5:00
